File: 240623/부분 문자열의 개수/number-of-substrings.py
a_str = input()
b_str = input()

a_len, b_len = len(a_str), len(b_str)

answer = 0
for i in range(a_len - b_len + 1):
    if a_str[i:i + b_len] == b_str:
        answer += 1

print(answer)

File: 251204/부분 문자열의 개수/number-of-substrings.cpp
#include <iostream>
#include <string>
using namespace std;

int main() 
{
    // Please write your code here.
    string a_str, b_str;
    int answer = 0, a_len, b_len;

    getline(cin, a_str);
    getline(cin, b_str);

    a_len = a_str.length();
    b_len = b_str.length();

    for(int i = 0; i < a_len - b_len + 1; i++)
    {
        if(a_str.substr(i, b_len) == b_str)
            answer += 1;
    }

    cout << answer;
    return 0;
}